front/app/modules/commercial/insights/admin/components/NavItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { NavItem } from 'containers/Admin/sideBar';
+import { FC, useEffect } from 'react';
+import { getUrlLocale } from 'services/locale';
+import { InsertConfigurationOptions } from 'typings';
+
+type Props = {
+  onData: (data: InsertConfigurationOptions<NavItem>) => void;
+};
+
+const NavItemComponent: FC<Props> = ({ onData }) => {
+  useEffect(
+    () =>
+      onData({
+        configuration: {
+          name: 'insights',
+          link: '/admin/insights',
+          iconName: 'processing',
```

Comment:
```suggestion
          iconName: 'processing',
          featureName: 'manual_insights',
```
Once the feature definition branch is merged, so the item only shows up when the feature is enabled. Since it's not there yet, that's not blocking, we just have to not forget in the end.

Reply:
Yes, indeed, I will make sure to keep this in mind and add it once we have the feature toggle 👍